options/locale/locale_cs-CZ.ini

```diff
@@ -221,7 +211,7 @@ invalid_db_setting=Nastavení databáze je neplatné: %v
 invalid_db_table=Databázová tabulka „%s“ je neplatná: %v
 invalid_repo_path=Kořenový adresář repozitářů není správný: %v
 invalid_app_data_path=Cesta k datům aplikace je neplatná: %v
-run_user_not_match="Run as" uživatelské jméno není aktuální uživatelské jméno: %s -> %s
+run_user_not_match="Run as\" uživatelské jméno není aktuální uživatelské jméno: %s -> %s
```

Comment:
See the comment in https://github.com/go-gitea/gitea/pull/23633#issuecomment-1480486999

So I decided not to touch it.

Reply:
If you prefer to manually fix these quotes, I can do it: in 72ec1092fc4708d5284eab49c7bcfa1c1722d51b , there are a lot. I can't do more at the moment because I can not sync crowdin

Many of these strings are still broken in 1.20, due to the mess of last restoring.